backend/projects/views/project.py

```diff
@@ -26,7 +26,7 @@ def get_permissions(self):
         return super().get_permissions()
 
     def get_queryset(self):
-        return Project.objects.filter(role_mappings__user=self.request.user)
+        return Project.objects.filter(role_mappings__user=self.request.user).order_by("created_at")
 
```

Comment:
Currently, the FE assumes that the list of projects and the list of project progress states are ordered in the same way. Hence, to ensure this, I think the same `.order_by("created_at")` needs to be added to the `get` method of class `ProjectProgressDetail`, on line 61. So something like:
```
all_projects = Project.objects.filter(role_mappings__user=self.request.user).order_by("created_at")
```